backend/api/views/tips.py

```diff
@@ -16,19 +21,166 @@ def get_tips():
     return create_response(data=response)

```

Comment:
do you still need this logger.info line?

Reply:
I could get rid of it, I didn't do logger in any of my other endpoints but for the get all tips I followed the structure of the other objects which had get all endpoints and they still had loggers in them. I assumed they were there for informative purposes, and since they were in the other endpoints I should do that in my tips endpoints too.